src/core/stats/remote-metrics.ts

```diff
@@ -72,8 +73,14 @@ export class RemoteMetrics {
     }, {} as Record<string, string>)
 
     formatted['library'] = 'analytics.js'
-    formatted['library_version'] =
-      `next-${process.env.VERSION}` ?? 'next-undefined'
+
+    if (typeof process.env.VERSION !== 'undefined') {
+      formatted['library_version'] =
+        `web:next-${version}` ?? 'web:next-undefined'
```

Comment:
let's remove the prefix for `web:` ... although cleaner, concerned about CDN users already relying on the format for something 😭